invisible_cities/cities/beersheba.py

```diff
@@ -86,14 +86,14 @@ def deconvolve_signal(det_db          : pd.DataFrame,
     iteration_tol   : Stopping threshold (difference between iterations).
     sample_width    : Sampling size of the sensors.
     bin_size        : Size of the interpolated bins.
-    energy_type     : Energy type ('E' or 'Ec', see Esmeralda) used for assignment.
-    deconv_mode     : 'joint' or 'separate', 1 or 2 step deconvolution, see description later.
```

Comment:
These are just changes from single forward quote to single back quote? I have no issue with the change, but it seems like a pretty minor detail.

Reply:
Single quotes in python mean it is a string, which is no longer the case. Backquotes are for formatting. x in backquotes means that I am referring to some variable named x in the code (and you see that github formats it in a monospace font like `x`).
In this docstring it is important to make this distinction, because up to now the config file needed to contain strings and from this PR on they will contain symbols, i.e. instead of `energy_type = 'E'`, the config file will read `energy_type = E`